Doc/library/sys.rst

```diff
@@ -1042,6 +1043,13 @@ always available.
       older Python versions include the version number, it is recommended to
       always use the ``startswith`` idiom presented above.
 
+   .. versionchanged:: 3.8
+      On AIX, :attr:`sys.platform` doesn't contain the major version anymore.
+      already uses the ``startswith`` idiom discussed above no changes to
+      existing code is expected. Remember, this is the version of AIX Python3
+      was built on, not running on. To get the version you are running on,
+      e.g., 6 or 7, use ``os.uname()[3]``
```

Comment:
Hum, this explanation is quite complicated. I would suggest to modify the idiom above to add AIX:

```

      if sys.platform.startswith('freebsd'):
          # FreeBSD-specific code here...
      elif sys.platform.startswith('linux'):
          # Linux-specific code here...
      elif sys.platform.startswith('aix'):
          # AIX-specific code here...
```

and copy/paste the Linux change:

```
      On AIX, :attr:`sys.platform` doesn't contain the major version anymore.
      It is always ``'aix'``, instead of ``'aix3'`` .. ``'aix7'``.  Since
      older Python versions include the version number, it is recommended to
      always use the ``startswith`` idiom presented above.
```

Reply:
OK. I was tempted to just copy/paste/substitute but the situation with AIX is not the same as it was with linux in 2011 as linux3 was becoming popular and there was a lot of code using sys.platform == 'linux2' (at least I read that 'between the lines').

The current practice is to use sys.platform.startswith('aix') because it is almost certain the number would not be correct. So, I felt silly 'recommending' something that is already common practice.

The value of this change is for the future - as it was nearly 7 years ago for linux.

If I do a search of all the code I see a mixture of sys.platform == 'linux' and sys.platform.startswith('linux') (and still one in test_util.py that sets sys.platform = 'linux2' (setting it, not testing it).

When I do the same for aix it always:
`find . | xargs grep sys.platform | grep aix`
it is either sys.platform.startswith("aix"), sys.platform.startswith('aix'), or  sys.platform[:3] == 'aix'.

As to AIX - this is the same for the (limited?) code review I have done of programs that have AIX specific sections.

Looking forward - code will be able to use sys.platform == 'aix'.

As to being 'complex' - my apologies. I tend to be verbose.

In any case, I'll make the changes as you request - but will also do them again, should that be needed. Getting the right balance in documentation is not simple.